playlistmanager.lua

```diff
@@ -1058,12 +1065,19 @@ mp.observe_property('playlist-count', "number", function(_, plcount)
   if playlist_visible then showplaylist() end
   if settings.prefer_titles == 'none' then return end
   -- resolve titles
-  resolve_titles()
+  if settings.resolve_url_titles and not settings.resolve_local_titles then
+    resolve_url_titles()
+  elseif settings.resolve_local_titles and not settings.resolve_url_titles then
+    resolve_local_titles()
+  elseif settings.resolve_url_titles and settings.resolve_local_titles then
+    resolve_url_titles()
+    resolve_local_titles()
```

Comment:
This can be simplified
```lua
if settings.resolve_url_titles then
    resolve_url_titles()
if settings.resolve_local_titles then
    resolve_local_titles()
end
```